docs/user-guide/howto.rst

```diff
@@ -157,6 +157,28 @@ one for each time bin.
 
 .. accordion-footer::
 
+.. accordion-header::
+    :id: collapseHowToOne
+    :title: Non-convergence of a fit because of IRFs validity range
```

Comment:
I am not sure the cause is the IRF validity range. It could happen with a dataset having PSF and edisp defined over the whole sky but a more limited exposure.

Reply:
The warning message is about "outside valid IRF map range". So the HowTo is about this specific case, only.
If the cause is not the the map range, then the warning message is bad! And one should change it.
Do you have any suggestion then?

Reply:
I think that the warning message is correct. It says that the model position is outside the IRF validity range, so the closest valid kernel is used. As @registerrier mentioned, the `npred` is zero because of no exposure